lib/grpc/transport/http2.ex

```diff
@@ -8,8 +8,12 @@ defmodule GRPC.Transport.HTTP2 do
 
   require Logger
 
+  def server_headers(%{codec: GRPC.Codec.WebText = codec}) do
+    %{"content-type" => "application/grpc-web-#{codec.name()}"}
+  end
+
   def server_headers(%{codec: codec}) do
-    %{"content-type" => "application/grpc+#{codec.name}"}
+    %{"content-type" => "application/grpc+#{codec.name()}"}
   end
```

Comment:
How do we assert on these in tests? I see no tests that ensure this part of the feature is working

Reply:
Added a test to ensure content types are set appropriately based on codec.